tools/quadrados/src/init.cpp

```diff
@@ -0,0 +1,134 @@
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+
+#include "tools.hpp"
+
+namespace fs = std::filesystem;
+
+static void printHelp()
+{
+    std::cerr << "Usage: quadrados init <PROJECT NAME> [OPTIONS]" << std::endl;
```

Comment:
If you can also change the other commands' help statements :relieved: 
```suggestion
    std::cerr << "Usage: quadrados [...] init <PROJECT NAME> [OPTIONS]" << std::endl;
```